### PR TITLE
Update Readme to fit latest changes in master

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
- Removed images, please comment if you still think it is necessary.
- Edited Prerequisites to only include necessary information. No need for installing dependencies manually anymore.
- Add comment about docker installation
- Removed section about how to get timers and alarms with a rest sensor in HA, not need anymore.
- Small fixes here and there

### CODE REVIEW

**Review by leikoilja — APPROVED** [review]
Looks really good! 💥  Thanks a lot! I have just added a small paragraph about HACS manual installation process for the early testers

File: README.md [hunk 1]
```diff
@@ -118,73 +92,15 @@ Please SSH to your system and run:
...
 
-Sensor state example:
+1. Restart Homeassistant after installation.
+2. In the Homeassistant UI go to "Configuration" -> "Integrations" click "+" and search for "Google Home".
 
-```yaml
-"entity_id": "sensor.google_nest_timer"
-"state": "[{'fire_time': 1611126011000.0, 'id': 'timer/617764ce-0000-2170-8ab3-2405887a817c', 'original_duration': 69062000.0, 'status': 1}]"
-```
+#### Running in Homeassistant Docker container
```
Comment (KapJI) [inline README.md:101]:
```suggestion
### Running in Homeassistant Docker container
```
It's not a subsection of "Integration Setup".

**Review by KapJI — COMMENTED** [review]
Manual installation steps can be simplified to:
1. `git clone https://github.com/leikoilja/ha-google-home.git`
2. `ln -s ha-google-home/custom_components/ha-google-home ~/.homeassistant/custom_components/ha-google-home`
3. Restard hass (old one)
4. Add integration (old one)

I think that's better than asking to "copy all files inside folder". For users with non standard directory for home assistant config you can add a small note.

**Comment by leikoilja** [comment]
@KapJI, thanks for the input! 💯 💥  
I have expanded the installation desc, let us know if you think we can improve it even further

**Comment by DurgNomis-drol** [comment]
@KapJI I forgot that, but not everyone knows how to use git. So i think we should a both method 😄 

And i can see that @leikoilja already did that 🚀

**Pull request merged**